### ISSUE TITLE
unable to reproduce the results on audio declipping task reported in paper

### ISSUE THREAD

**Issue by QinpengCui** [YWolfeee/Training-Free-Guidance#6]
Thank you for this remarkable work!
When I test TFG on the audio declipping task, it produces the following results:

Logging to logs/audio_declipping/tfg+recur_steps=1+iter_steps=4/target=no/rho=1.0-increase+mu=1.0-increase+sigma=0.1-decrease
Arguments: Arguments(data_type='audio', dataset='teticio/audio-diffusion-256', task='audio_declipping', image_size=256, include_charges=False, generators_path='./pretrained_models/EDMsecond/generative_model_ema.npy', args_generators_path='./pretrained_models/EDMsecond/args.pickle', energy_path='./pretrained_models/tf_predict_mu/model_ema_2000.npy', args_energy_path='./pretrained_models/tf_predict_mu/args_2000.pickle', classifiers_path='./pretrained_models/evaluate_mu/best_checkpoint.npy', args_classifiers_path='./pretrained_models/evaluate_mu/args.pickle', clip_scale=100, audio_length=10, volume_factor=80.0, motion_prompt='walk', radius=3.0, model_name_or_path='audio-diffusion-256', train_steps=1000, inference_steps=100, eta=1.0, clip_x0=True, clip_sample_range=1.0, seed=42, device=device(type='cuda'), logging_dir='logs/audio_declipping/tfg+recur_steps=1+iter_steps=4/target=no/rho=1.0-increase+mu=1.0-increase+sigma=0.1-decrease', logger=None, per_sample_batch_size=8, num_samples=256, batch_id=0, guidance_name='tfg', guider='classifier', target='no', recur_steps=1, iter_steps=4, guidance_strength=8.0, rho=1.0, mu=1.0, sigma=0.1, eps_bsz=1, rho_schedule='increase', mu_schedule='increase', sigma_schedule='decrease', guide_network='no', classifier_image_size=224, eval_batch_size=16, logging_resolution=512, log_suffix='', log_traj=False, max_show_images=256, check_done=False, wandb=False, wandb_project='trail', wandb_name=None, wandb_entity='llm-selection', saved_file=None, sort_metric=None, topk=5, output_path='vis_molecule', max_n_samples=10000000000)
audio samples are saved in logs/audio_declipping/tfg+recur_steps=1+iter_steps=4/target=no/rho=1.0-increase+mu=1.0-increase+sigma=0.1-decrease/audios
Evaluating 256 samples
{'recover_score': -24.352653304259437, 'fad': 0.6809635173891131}

This seems to differ significantly from the metrics reported in the paper. Is there any issue causing this?
